src/scripts.js

```diff
@@ -1,12 +1,16 @@
-const userMain = document.querySelector('.userMain')
-const userProfile = document.querySelector('.userProfile')
+const userMain = document.querySelector('.user-main')
+const userProfile = document.querySelector('.user-profile')
 const friends = document.querySelector('.friends')
-const userStepGoal = document.querySelector('.userStepGoal')
-const communityStepGoal = document.querySelector('.communityStepGoal')
-
+const userStepGoal = document.querySelector('.user-step-goal')
+const communityStepGoal = document.querySelector('.community-step-goal')
+const water = document.querySelector('.water');
+const waterStats = document.querySelector('.water-stats');
```

Comment:
It looks like the class water-stats is not yet defined in our HTML, so referencing this is giving a 'null' error.

Reply:
@srslie Yes, thanks for catching that!!